.golangci.yml

```diff
@@ -73,7 +73,6 @@ linters:
     # https://github.com/1uf3/execinquery is archived so golangci-lint warns/errors about it
     # - execinquery # execinquery is a linter about query string checker in Query function which reads your Go src files and warning it finds
     - exhaustive # check exhaustiveness of enum switch statements
-    - exportloopref # checks for pointers to enclosing loop variables
```

Comment:
```
level=warning msg="The linter 'exportloopref' is deprecated (since v1.60.2) due to: Since Go1.22 (loopvar) this linter is no longer relevant. Replaced by copyloopvar."
```